src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php

```diff
@@ -1337,4 +1337,9 @@ protected function wrapJsonSelector($value)
 
         return 'json_unquote(json_extract('.$field.$path.'))';
     }
+
+    public function compileConnectionCount()
+    {
+        return 'show status where variable_name = "threads_connected"';
```

Comment:
```suggestion
        return 'select variable_value as `Value` from performance_schema.session_status where variable_name = "threads_con
nected"'
```

Reply:
This is different from the [current query for MySQL](https://github.com/laravel/framework/pull/52231/files#diff-9818924e4ab4c2527e02edf7ac598ff8593bcf13198b0ddeabf1762c8cc1c38cL45).